roles/deploy/tasks/main.yml

```diff
@@ -241,25 +245,25 @@
 
   # The dev role installed nvm (node version manager) for nodejs
   # Read the NODE_VERSION form component_info (if any)
-  - name: Fetch nodejs version from component_info
+  - name: DEVELOP - Fetch nodejs version from component_info
```

Comment:
Nitpick: good fix, adding DEVELOP here and fixing typo's. But should not be part of this PR. It makes it less obvious what was changed.